src/IAWebRedirectSide/IAWebRedirectSide.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import '../Forside/Forside.less';
+import { RestSykefraværshistorikk } from '../api/sykefraværshistorikk';
+
+interface Props {
+    restSykefraværshistorikk: RestSykefraværshistorikk;
+}
+
+const IAWebRedirectSide: React.FunctionComponent<Props> = props => {
+    {
+        return (
+            <div className="forside__wrapper">
```

Comment:
Er vel ikke helt riktig å kalle denne for "forside"

Reply:
Vi bruker samme klasse, litt i tvil om det. men ville ikke duplisere kode :)

Reply:
Enig i det, men da burde vi løse det ved å lage en egen komponent vi kan bruke, eller exponere en mixin.
Hvis vi endrer stylingen på forsiden nå, så kommer denne siden til å bli påvirket også - og det tenker vi helt sikkert ikke på, når vi bare styler Forside.less 🙂